tests/containers/helm.pm

```diff
@@ -16,21 +16,43 @@
 
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use utils qw(script_retry);
+use utils qw(zypper_call script_retry);
+use version_utils qw(is_sle);
 use mmapi 'get_current_job_id';
+use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::k8s;
 
 sub run {
-    my ($self) = @_;
+    my ($self, $run_args) = @_;
+
+    my $is_k3s = $run_args->{k8s_env} eq 'k3s';
+    my $is_eks = $run_args->{k8s_env} eq 'eks';
+    $self->{is_k3s} = $is_k3s;
 
     $self->select_serial_terminal;
     $self->{deployment_name} = "apache-" . get_current_job_id();
     my $chart = "bitnami/apache";
 
-    install_k3s();
-    install_kubectl();
+    record_info("Kubernetes environment", $run_args->{k8s_env});
+    record_info("Chart name", $chart);
+
+    install_k3s() if $is_k3s;
+    install_kubectl() if $is_eks;
```

Comment:
can this be moved below under L#43? It would be nice to have as much as `eks` stuff under this condition

Reply:
This has changed